natlas-server/app/__init__.py

```diff
@@ -6,7 +6,7 @@
 from flask_wtf.csrf import CSRFProtect
 from config import Config, populate_defaults, get_defaults
 from app.elastic import ElasticInterface
-from app.scope import ScopeManager
+from app.scope.scope import ScopeManager
```

Comment:
You should create an `__init__.py` file inside `app/scope` and import ScopeManager in that file, which then provides access to ScopeManager from `app.scope` instead of requiring `app.scope.scope`. This is also helpful if we want to provide other public components of the module in the future beyond just the ScopeManager entity. It puts the module in control of what it exports and makes you have to do additional work to export anything else.